### PR TITLE
Update doc on `mailcacher` usage

### PR DESCRIPTION
**Pull request by davidlukac**

### CODE REVIEW

File: mailcatcher/README.md [hunk 1]
```diff
@@ -6,3 +6,30 @@ FROM quay.io/continuouspipe/mailcatcher:stable
...
+
+```
+mailcatcher:
+    image: quay.io/continuouspipe/mailcatcher:stable
+    extra_hosts:
+        - "mailcatcher:127.0.0.1"
+    expose:
+        - 25
+    ports:
+        - "1080:80"
```
Comment (andytson-inviqa) [inline mailcatcher/README.md]: the ports are 1025 and 1080 on the container

File: mailcatcher/README.md [hunk 2]
```diff
@@ -6,3 +6,30 @@ FROM quay.io/continuouspipe/mailcatcher:stable
...
+
+Then in `parameters.yml` (in case of Symfony application):
+
+```
+mailer_transport: smtp
+    mailer_host: mailcatcher
+    mailer_user: null
+    mailer_password: null
+    mailer_sender_mail: some@email.com
+    mailer_port: 25
```
Comment (andytson-inviqa) [inline mailcatcher/README.md]: 1025

File: mailcatcher/README.md (hunk 1 above)
Comment (davidlukac) [inline mailcatcher/README.md]: well, those don't work :-)
from the log:
```
mailcatcher_1  | Starting MailCatcher
mailcatcher_1  | ==> smtp://0.0.0.0:25
mailcatcher_1  | ==> http://0.0.0.0:80
```
Once they do I'll be happy to update the usage doc :)

File: mailcatcher/README.md (hunk 2 above)
Comment (davidlukac) [inline mailcatcher/README.md]: nope, doesn't work. Either dockerfile needs fixing, so those ports are actually exposed on 1025 and 1080, or the we remove it from Dockerfile.

Comment (kierenevans) [inline mailcatcher/README.md]: I believe this is a difference between :latest and :stable tags. Going to push a new :stable version now.

Comment (davidlukac) [inline mailcatcher/README.md]: 👍  great, let me know and I'll re-test with the higher ports

Comment (kierenevans) [inline mailcatcher/README.md]: Done, if testing locally please do `docker-compose pull mailcatcher` and then `docker-compose up mailcatcher` to redeploy.

**Comment by davidlukac** [comment]
Latest version works well with the higher ports, doc updated accordingly.

**Comment by sroze** [comment]
Thanks @davidlukac !

**Pull request merged**